README.md

```diff
@@ -304,120 +304,11 @@ export He_AcrPassword=$(az keyvault secret show --vault-name $He_Name --name "Ac
-
-```bash
-
-# Add Log Analytics extension
-az extension add -n log-analytics
-
-# create Log Analytics for the webv clients
-az monitor log-analytics workspace create -g $He_WebV_RG -l $He_Location -n $He_Name -o table
-
-# retrieve the Log Analytics values using eval $He_LogAnalytics_*
-export He_LogAnalytics_Id='az monitor log-analytics workspace show -g $He_WebV_RG -n $He_Name --query customerId -o tsv'
-export He_LogAnalytics_Key='az monitor log-analytics workspace get-shared-keys -g $He_WebV_RG -n $He_Name --query primarySharedKey -o tsv'
-
-# save the environment variables
-./saveenv.sh -y
-
-# create Azure Container Instance running webv
-az container create -g $He_WebV_RG --image retaildevcrew/webvalidate:debug -o tsv --query name \
--n ${He_Name}-webv-${He_Location} -l $He_Location \
---log-analytics-workspace $(eval $He_LogAnalytics_Id) --log-analytics-workspace-key $(eval $He_LogAnalytics_Key) \
---command-line "dotnet ../webvalidate.dll --tag $He_Location -l 1000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --json-log"
-
-# create in additional regions (optional)
-az container create -g $He_WebV_RG --image retaildevcrew/webvalidate:debug -o tsv --query name \
--n ${He_Name}-webv-eastus2 -l eastus2 \
---log-analytics-workspace $(eval $He_LogAnalytics_Id) --log-analytics-workspace-key $(eval $He_LogAnalytics_Key) \
---command-line "dotnet ../webvalidate.dll --tag eastus2 -l 10000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --json-log"
-
-az container create -g $He_WebV_RG --image retaildevcrew/webvalidate:debug -o tsv --query name \
--n ${He_Name}-webv-westeurope -l westeurope \
---log-analytics-workspace $(eval $He_LogAnalytics_Id) --log-analytics-workspace-key $(eval $He_LogAnalytics_Key) \
---command-line "dotnet ../webvalidate.dll --tag westeurope -l 10000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --json-log"
-
-az container create -g $He_WebV_RG --image retaildevcrew/webvalidate:debug -o tsv --query name \
--n ${He_Name}-webv-southeastasia -l southeastasia \
---log-analytics-workspace $(eval $He_LogAnalytics_Id) --log-analytics-workspace-key $(eval $He_LogAnalytics_Key) \
---command-line "dotnet ../webvalidate.dll --tag southeastasia -l 10000 -s https://${He_Name}.azurewebsites.net -u https://raw.githubusercontent.com/retaildevcrews/${He_Repo}/master/TestFiles/ -f benchmark.json -r --json-log"
-
-```
-
-### Sample Queries
-
-Click on the 'Logs' item in the Log Analytics sidebar menu and run the `ContainerInstanceLog_CL` query to view all Azure Container Instance logs. Note that it may take several minutes after the ACI creation for logs to populate in Log Analytics.
```

Comment:
Removed the line where it can take several minutes for logs to appear. Posted my findings about this in the parent issue.